pkg/virtual/apiexport/schemas/builtin/builtin.go

```diff
@@ -261,6 +272,16 @@ var BuiltInAPIs = []internalapis.InternalAPI{
 		Instance:      &admissionregistrationv1alpha1.ValidatingAdmissionPolicyBinding{},
 		ResourceScope: apiextensionsv1.ClusterScoped,
 	},
+	{
+		Names: apiextensionsv1.CustomResourceDefinitionNames{
+			Plural:   "validatingadmissionpolicybindings",
```

Comment:
I definitely want an e2e test for this (there's a note on it in the PR description), but I didn't want to squeeze it into this PR. Do you think it would be better to do this immediately?

Reply:
If the API is enabled, we should test it. We can disable it and postpone to followup.

Reply:
Test has been added. It was in fact not working, I had to fix things on k8s and kcp side. Key commits:

- https://github.com/kcp-dev/kubernetes/commit/d61cdaca2a4ad0b8e57f31325f3aaf4c507bfec6
- https://github.com/kcp-dev/kubernetes/commit/3b450c7d0daf983858646073b08aca3e0ed9ba9e 
- https://github.com/kcp-dev/kcp/pull/3140/commits/bcf32edaa5456d0bc9e5208534a17313e3793154
- https://github.com/kcp-dev/kcp/pull/3140/commits/6af5c9314b07d08b0962ba11599779c8df179e49